.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,45 @@
+---
+name: Bug report
+about: Create a bug report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Expected Behavior
+<!--- Tell us what should happen -->
+
+## Current Behavior
+<!--- Tell us what happens instead of the expected behavior -->
+
+## Possible Solution
+<!--- Not obligatory, but suggest a fix/reason for the bug, -->
+
+## Context
+<!--- How has this issue affected you? What are you trying to accomplish? -->
+<!--- Providing context helps us come up with a solution that is most useful in the real world -->
+
+## Steps to Reproduce
+<!--- Provide a link to a live example, or an unambiguous set of steps to -->
+<!--- reproduce this bug. Include code to reproduce, if relevant -->
+1.
+2.
+3.
+4.
+
+## Your Environment
+<!--- Include as many relevant details about the environment you experienced the bug in -->
+* Bash-it version used:
+* List of enabled plugins, themes and aliases:
```

Comment:
Maybe add here to use `bash-it show plugins` etc.

Reply:
Yea, I should also add `bash-it doctor` output request

Reply:
Maybe we should include a short summary of enabled components as part of `bash-it doctor`. So instead of having to run several commands, we just run one command to produce the required output...

The list of components could be condensed to only show the enabled ones on one line (one for aliases, one for completions, one for plugins), instead of the line-by-line output that `bash-it show` currently produces.

Reply:
made #1745 for that